sources/packages/web/src/views/aest/institution/ApplicationOfferingChangeRequestForm.vue

```diff
@@ -201,6 +214,8 @@ export default defineComponent({
           snackBar.error(
             "Unexpected error while submitting application offering change request.",
           );
+        } finally {
+          assessApplicationOfferingChangeRequestModal.value.hideModal();
```

Comment:
Is it not the case to leave the modal open in case of an error to allow the user to try again?
The modal should be closed probably when everything goes well.
In case of an error happen probably the loading should probably be set to false only. Does it make sense?